sw/silta/stm32f407.py

```diff
@@ -93,11 +117,13 @@ class bridge(Silta):
     __SPI_MAX_BYTES = 1024
     __I2C_MAX_BYTES = 1024
 
+    PIN = [1 << i for i in range(15)]
```

Comment:
I can't see where this gets used, but it looks new in your change. Maybe my eyes just aren't catching it?